lib/defaults.js

```diff
@@ -10,28 +11,35 @@ module.exports = {
   email: {
     sender: '"🤖 Eos - Github Bot" <noreply@md.getsentry.com>',
     subjectTemplate: 'Attention: <%= count %> Pull requests',
```

Comment:
The subject doesn't fit that well anymore.
How about `Hey! Github needs your attention`